Makefile

```diff
@@ -1,7 +1,25 @@
+APP_NAME = ups-sidecar
```

Comment:
should we leave that name for now?

Reply:
yeah - for now, that's fine.  Or we can roll the dice :game_die: on something more fancy tomorrow :-)  #fruehschoppen